.babelrc

```diff
@@ -34,7 +34,10 @@
     ["@babel/plugin-proposal-class-properties", { "loose": false }],
     "@babel/plugin-proposal-json-strings",
     ["module-resolver", {
-      "root": "./src"
+      "root": "./src",
+      "alias": {
+        "vet360": "./src/platform/user/profile/vet360"
```

Comment:
I kinda love how clean this makes the imports. `...from 'vet360/constants'` is so much nicer than `...from 'platform/user/profile/vet360/constants'`

Reply:
That's cool. Didn't know you can do this.